### ISSUE TITLE
Unable to inspect new window in windows desktop app

### ISSUE THREAD

**Issue by jbradstreet** [appium/appium-desktop#372]
## Appium or Appium Desktop?

You are reporting an issue at the Appium Desktop repository. Appium Desktop is a wrapper around Appium. If you are having trouble running tests, it is much more likely that the problem you are encountering is not a problem with Appium Desktop but with Appium. For that reason we require you to have tried your testcase with Appium CLI before reporting issues. Check the checkbox below to confirm that you have proven your issue does _not_ reproduce on Appium itself:

- [ ] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

I'm trying to automate a test for a windows desktop app - Outlook. When using the appium inspector, I am unable to target a new pop up window within Outlook. Screen recording here: https://cl.ly/0r043K2i0U3q

Even when I refresh the source image, it keeps pulling the first window to the foreground and using that...instead of the second one (which I want/need to continue writing my test). 

Anyone know how to target a new pop up window using the inspector? 

## Environment

- I am running Appium Desktop version _<insert version here>_.
- I am on (pick one):
    - [ ] Mac
    - [ ] Windows
    - [ ] Linux

**Comment by dpgraham** [comment]
Yeah it only inspects the original window. I'll have to look into seeing if there's a way to switch windows.

**Comment by rekha110254** [comment]
Unable to inspect elements inside FlatList. i'm using 1.7.2 (Mac)

**Comment by junllfernandez** [comment]
Having the same issue and I'm currently using Appium Desktop version 1.9.0. I'm testing the same windows desktop app - Outlook. Any solution on this issue?

Thanks!
